guides/common/modules/proc_configuring-satellite-with-an-alternate-cname.adoc

```diff
@@ -0,0 +1,16 @@
+[id='configuring-satellite-with-an-alternate-cname_{context}']
```

Comment:
is it intentional that the ID and the filename have "satellite" in them?

Reply:
Well, this is consistent, as this is done throughout the guides. For now, let\s stick to consistency, and if we want to change it in future we can do it everywhere in one run.

Reply:
Works for me, was mostly curious :)

Reply:
So far we have removed the word from directory names. I suggest to make the switch rather earlier. This will ultimately break links in downstream tho.

By the way can't we simply create TWO ids (archors)? One with satellite for backward compatibility and one with foreman (or with something else like "project" or whatever.

Reply:
Two IDs will be complicated to maintain and create garbage in files. I will substitute `satellite` with `project` in the filename and in ID here.